src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -184,13 +174,7 @@ Napi::Value CoreWrap::compile_model_sync_dispatch(const Napi::CallbackInfo& info
             return compile_model_sync(info, info[0].ToObject(), info[1].ToString(), config);
         }
 
-        std::string arguments_error_message =
-            std::accumulate(errors_messages.begin(),
-                            errors_messages.end(),
-                            std::string(),
-                            [](const std::string& a, const std::string& b) -> std::string {
-                                return a.empty() ? b : a + "\nor\n " + b;
-                            });
+        std::string arguments_error_message = NapiArg::join_array_to_str(errors_messages, "\nor\n ");
```

Comment:
```suggestion
        auto arguments_error_message = ov::util::join(errors_messages, "\nor\n ");
```
I think should give same result.